build.gradle

```diff
@@ -62,6 +62,7 @@ buildscript {
     classpath "com.github.jengelman.gradle.plugins:shadow:2.0.1"                                        // Enable shading Java dependencies
     classpath "ca.coglinc:javacc-gradle-plugin:2.4.0"                                                   // Enable the JavaCC parser generator
     classpath "gradle.plugin.io.pry.gradle.offline_dependencies:gradle-offline-dependencies-plugin:0.3" // Enable creating an offline repository
+    classpath "net.ltgt.gradle:gradle-errorprone-plugin:0.0.13"
```

Comment:
Add comment stating what the plugin does, its not always obvious like this one is but having the comment will make it consistent for where people look.

Reply:
Done